rootfs-builder/README.md

```diff
@@ -0,0 +1,57 @@
+# Building a rootfs for Kata Containers Guest OS #
+
+Kata Containers guest OS is created using `rootfs.sh`.
+
+## Supported base OS ##
+
+The `rootfs.sh` script support different based OS `rootfs`. To build
+a `rootfs` based in a supported based OS run:
+
+``` ./rootfs.sh <distro> ```
+
+To check the supported `rootfs` based OS use:
+
+```
+./rootfs.sh -h
+```
+
+## Adding support for new base OS ##
+
+The script `rootfs.sh` will look for directories that meets the following
```

Comment:
Sorry to be super-pedantic but when you say "look" do you mean it will do a recursive `find`? Or will it check for immediate sub-directories containing the expected files?